src/scripts/lib/server/pipeline/index.js

```diff
@@ -15,7 +15,7 @@ const flatten = function(array) {
 
 module.exports = class Pipeline {
 
-  static DATA_DIR = 'data'
+  static DATA_DIR = '@@clientData'
```

Comment:
This will always execute in a Node.js context and won't be included in the client bundle, so no replacement is necessary, just reading it directly from the config store.